src/views/TrialDescriptionPage/TrialDescriptionPage.jsx

```diff
@@ -2,15 +2,16 @@ import queryString from 'query-string';
 import React, { useState, useEffect } from 'react';
 import { Helmet } from 'react-helmet';
 import { useDispatch, useSelector } from 'react-redux';
-import { useLocation, useNavigate } from 'react-router-dom';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
 import { useTracking } from 'react-tracking';
+import { START_OVER_LINK } from '../../constants';
 
 import {
 	Accordion,
 	AccordionItem,
 	Delighter,
 	TrialStatusIndicator,
-	SearchCriteriaTable,
+	SearchCriteriaTableUpdated,
```

Comment:
Shouldn't this rather be a filename change to replace the old one rather than maintaining this name `SearchCriteriaTable` rather than `SearchCriteriaTableUpdated`.

Reply:
Since I have another ticket to replace SCT on results page I need to have both files for now 
We can have a separate issue created to delete old table and rename the updated one back to being just SearchCriteriaTable

Reply:
> 

Ah, yasna!